test/server/index.js

```diff
@@ -160,6 +162,31 @@ app.post('/streams/in', function(req, res) {
   })
 });
 
+app.get('/send_cookie', function(req, res) {
+  if (req.cookies === undefined || Object.keys(req.cookies).length === 0) {
+    res.cookie('connected', 'blue');
+	res.status(200).send('New cookie created. Returning it now.');
+  } else {
+	var connected = req.cookies['connected'];
+	res.status(200).send('Cookie already existed: ', connected, '.',
+	                     '(This is OK.) Returning it now.');
+  }
+});
+
+app.get('/receive_cookie', function(req, res) {
```

Comment:
Change the name of this endpoint to "verify_cookie". As for the other, add a query parameter which shall be the name of the cookie you expect to find.

Reply:
DONE.